tautulli/tools/webhooks/discord.py

```
@@ -33,18 +37,64 @@
+    content: bytes
+
+    @classmethod
+    def from_flask_request_files(cls, files: dict) -> List["DiscordWebhookAttachment"]:
+        """
+        Ingest a list of files from a Discord webhook Flask request
+        :param files: The files from the Flask request
+        :return: A list of Discord webhook attachments
+        """
+        # Will probably only have one key, 'files[0]'
+        # ref: https://github.com/Tautulli/Tautulli/blob/d019efcf911b4806618761c2da48bab7d04031ec/plexpy/notifiers.py#L1170
+        for key, value in files.items():
```

Ruff (0.8.0)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)